packages/unified-latex-util-parse/tests/parse-basic.test.ts

```diff
@@ -138,4 +140,36 @@ describe("unified-latex-util-parse", () => {
         parsed = parseMath("\\x   #1    y");
         expect(printRaw(parsed)).toEqual("\\x #1 y");
     });
+
+    it("Does not parse verbatim in listings and minted packages", () => {
+        expect(trimRenderInfo(parse("\\lstinline{some_code$}"))).toEqual({
+            type: "root",
+            content: [m("lstinline", args([null, "some_code$"], { braces: "[]{}" }))],
+        });
+        expect(trimRenderInfo(parse("\\lstinline[language]{some_code$}"))).toEqual({
+            type: "root",
+            content: [m("lstinline", args(["language", "some_code$"], { braces: "[]{}" }))],
+        });
+        expect(trimRenderInfo(parse("\\lstinline#some_code$#"))).toEqual({
+            type: "root",
+            content: [m("lstinline", [arg(null), arg("some_code$", { openMark: "#", closeMark: "#" })])],
+        });
+        expect(trimRenderInfo(parse("\\lstinline[language]#some_code$#"))).toEqual({
+            type: "root",
+            content: [m("lstinline", [arg("language", { braces: "[]" }), arg("some_code$", { openMark: "#", closeMark: "#" })])],
+        });
+        expect(trimRenderInfo(parse("\\lstinline[foo %bar\n\n]{my code}"))).toEqual({
+            type: "root",
+            content: [m("lstinline", [arg([s("foo"), { type: "comment", content: "bar", leadingWhitespace: true, sameline: true, suffixParbreak: true } , { type: "parbreak" }], { braces: "[]" }), arg("my code")])],
+        });
+
+        expect(trimRenderInfo(parse("\\mint[options]{some language}#some_code$#"))).toEqual({
+            type: "root",
+            content: [m("mint", [...args(["options", "some language"], { braces: "[]{}" }), arg("some_code$", { openMark: "#", closeMark: "#" })])],
+        });
+        expect(trimRenderInfo(parse("\\mintinline[options]{some language}#some_code$#"))).toEqual({
+            type: "root",
+            content: [m("mintinline", [...args(["options", "some language"], { braces: "[]{}" }), arg("some_code$", { openMark: "#", closeMark: "#" })])],
+        });
+    });
```

Comment:
These tests look great! Can you add a couple more with bad behavior? Like putting a `%` in the verbatim content and perhaps some newlines and some random back-slashes.

Reply:
Done!